tests/backends/test_native_backend_against_coincurve.py

```diff
@@ -31,12 +31,12 @@
 MAX_EXAMPLES = 200
 
 
-@pytest.fixture
+@pytest.fixture(scope="module")
 def native_key_api():
     return KeyAPI(backend=NativeECCBackend())
 
 
-@pytest.fixture
+@pytest.fixture(scope="module")
```

Comment:
Why this change?

Reply:
There is a hypothesis deprecation warning that says: 
```
<test name> uses the coincurve_key_api fixture, which is reset between function calls 
but not between test cases generated by '@given(...)'.  
You can change it to a module- or session-scoped fixture if it is safe to reuse; 
if not we recommend using a context manager inside your test function.  
See https://docs.pytest.org/en/latest/fixture.html#sharing-test-data for details on fixture scope.
```
So I decided to make that change in this PR. Happy to pull it out into it's own PR if you have a preference.